FMS/Pages/Shared/_Layout.cshtml

```diff
@@ -1,54 +1,60 @@
-<environment exclude="Production">
-    <div class="banner-test-site">
-        Test Site
-    </div>
-</environment>
-<a class="skipnav" href="#main-content">Skip to main content</a>
-<header role="banner">
-    <div class="container h3 my-2">
-        <img src="~/images/epd-icon.svg" alt="Georgia EPD" class="img-logo" />
-        <a href="/" aria-label="Home" class="text-dark">File Management System</a>
-    </div>
-    @if (User.Identity?.IsAuthenticated ?? false)
-    {
-        <partial name="_MenuPartial" />
-    }
-</header>
-<div class="container">
-    <main id="main-content" role="main" class="pb-3">
-        @RenderBody()
-    </main>
-</div>
-
-<footer class="border-top footer">
-    <div class="container text-brand">
-        <a href="#">Return to top</a>
-    </div>
-    <div class="bg-brand text-light">
-        <div class="container">
-            &copy; @DateTime.Now.Year Environmental Protection Division IT -
-            <a href="https://georgia.gov/accessibility" target="_blank" rel="noopener">Accessibility</a> |
-            <a href="https://gadnr.org/ada" target="_blank" rel="noopener">DNR Accessibility Notice</a> |
-            <a href="https://georgia.gov/privacysecurity" target="_blank" rel="noopener">Privacy/Security</a>
-        </div>
-    </div>
-</footer>
-
-<script src="~/lib/jquery/jquery.min.js"></script>
-<script src="~/lib/bootstrap/js/bootstrap.bundle.min.js"></script>
-@await RenderSectionAsync("Scripts", required: false)
-</body>
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FMS - @ViewData["Title"]</title>
+    <partial name="_FaviconPartial" />
+    <link rel="stylesheet" href="~/lib/bootstrap/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="~/css/site.css" />
+    @await RenderSectionAsync("Styles", required: false)
+    <partial name="_RaygunScriptsPartial" />
+</head>
+
+<body id="top" class="d-flex flex-column">
+    <environment exclude="Production">
+        <div class="banner-test-site">
+            Test Site
+        </div>
+    </environment>
+    <a class="skipnav" href="#main-content">Skip to main content</a>
+    <header role="banner">
+        <div class="container h3 my-2">
+            <img src="~/images/epd-icon.svg" alt="Georgia EPD" class="img-logo" />
+            <a href="/" aria-label="Home" class=" text-dark text-decoration-none">File Management System</a>
+        </div>
+        @if (User.Identity?.IsAuthenticated ?? false)
+        {
+            <partial name="_MenuPartial" />
+        }
+    </header>
+
+    <main id="main-content" role="main" class="pb-3">
+        <div class="container">
+            @RenderBody()
+        </div>
+    </main>
+
+    <footer class="footer pt-3">
+        <div class="container border-bottom text-brand py-2">
+            <a href="#">Return to top</a>
+        </div>
+        <div class="bg-brand text-light pb-3">

```

Comment:
The changes you've made to the footer spacing are causing it to overlap the body element. For example, I can't click the bottom half of the Create New Facility button:

![image](https://github.com/gaepdit/FMS/assets/737326/6cdc08f7-8ce7-4eb9-818c-f32143e6e7e9)

Currently, the body is given [120px of space for the footer](https://github.com/gaepdit/FMS/blob/a5b7d582769cc214df9c0a0c38cbe9bb2bc9a0e7/FMS/wwwroot/css/site.css#L185-L187). Either increase the bottom margin on the body or decrease the footer height.